arch/arm/core/irq_manage.c

```diff
@@ -84,19 +84,18 @@ int _arch_irq_is_enabled(unsigned int irq)
  */
 void _irq_priority_set(unsigned int irq, unsigned int prio, u32_t flags)
 {
-	/* Hardware priority levels 0 and 1 reserved for Kernel use.
-	 * So we add 2 to the requested priority level. If we support
-	 * ZLI, 2 is also reserved so we add 3.
+	/* Hardware priority levels can be reserved for Kernel use.
+	 * So we add to the requested priority level.
 	 */
 
 #if CONFIG_ZERO_LATENCY_IRQS
-	/* If we have zero latency interrupts, that makes priority level 2
+	/* If we have zero latency interrupts, the last priority level is
```

Comment:
Just to clarify: last priority level == the lowest-level priority?

Reply:
I'll update the comment along with the use of the define to clarify.
I meant to say the lowest priority level of the reserved ones.